.github/workflows/s3-to-github.yml

```
@@ -20,11 +20,14 @@ name: Sync S3 to GitHub
 on: # yamllint disable-line rule:truthy
   workflow_dispatch:
     inputs:
-      source-location:
+      destination-location:
```

```suggestion
      source-location:
```
source is s3 for s3 to github

---

:facepalm: Right :)